pkg/query/querier_test.go

```diff
@@ -274,6 +277,54 @@ func TestDedupSeriesIterator(t *testing.T) {
 	}
 }
 
+func TestQuerier_StoreReadTimeout(t *testing.T) {
+	storeClient := &store.ClientMock{
```

Comment:
I would be more comfortable if this test would have also another client that actually serves some metrics after 100 miliseconds.